src/huggingface_hub/keras_mixin.py

```diff
@@ -39,10 +39,13 @@ def _extract_hyperparameters_from_keras(model):
     return hyperparameters
 
 
-def _parse_model_history(model):
+def _parse_model_history(model, save_directory):
     lines = None
     if model.history is not None:
         if model.history.history != {}:
+            path = os.path.join(save_directory, "history.json")
+            with open(path, "w") as f:
+                json.dump(model.history.history, f)
```

Comment:
Small nits here for Hugging Face style points. 🤗

```suggestion
            with open(path, "w", encoding="utf-8") as f:
                json.dump(model.history.history, f, indent=2, sort_keys=True)
```

For reference...

[Here's](https://github.com/huggingface/transformers/blob/31ec2cb2badfbdd4c1ac9c6c9b8a74e974984206/src/transformers/configuration_utils.py#L792) where I got the style from


And [here's](https://github.com/huggingface/transformers/blob/31ec2cb2badfbdd4c1ac9c6c9b8a74e974984206/src/transformers/configuration_utils.py#L805-L806) why I went with specifying the encoding explicitly (I really dunno if this is necessary, but it doesn't hurt I suppose 😅 )

Feel free to ignore if you disagree...I just think its easier to read on the Hub when its formatted all pretty 💄

Reply:
Totally agree, git diffs will also be better if JSON is pretty formatted. Good catch @nateraw